eclipse-theia/theia
# [example apps] update html-language-features vscode built-in

#### What it does
We discovered that this built-in extension requires to have typescript bundled, to handle HTML-embedded JS/TS.

This PR updates the vscode built-ins so that a fixed version of that extension is used. I updated the html built-in too, just in case.

See upstream issue: https://github.com/theia-ide/vscode-builtin-extensions/issues/33

#### How to test


_update: if testing without this PR to confirm the bug, make sure to do so in a new, empty workspace. This should avoid the workspace's `typescript` being used, giving you LS support without this fix. Also make sure you do not have `typescript` installed globally since then that one will probably be used. Finally I think maybe typescript from builtin `typescript-language-features` might be used, if nothing else is found_  

Conclusion: it can be tricky to reproduce the initial bug using our example application - it has multiple options to find a `typescript` to use. 

Open a `.html` file that contains embedded JS/TS in the example application and verify that you get TypeScript LS features in the editor. e.g. upon hovering on JS/TS elements, the LS should provide information about them.

Before this change, an error about `typescript` not being found would be triggered. 

If you do not have such an HTML file handy, you can use the content of this one here: 
https://github.com/microsoft/vscode/blob/master/test/unit/electron/renderer.html

#### Review checklist


#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
